app/views/templates/_article.html.erb

```diff
@@ -41,24 +41,27 @@
 
       <div class="article__buttons">
         <% if cms_block_content(:file_1)&.file&.present? %>
-          <% file_1_language  = cms_block_content(:file_1_language) || "English" %>
-          <% file_1_label     = cms_block_content(:file_1_label) || "Download file" %>
           <% file_1           = cms_block_content(:file_1).file %>
+          <% file_1_language  = cms_block_content(:file_1_language) || "English" %>
+          <% file_1_label     = cms_block_content(:file_1_label) || file_name_for(file_1) %>
 
           <p>
-            <strong>File name:</strong> <%= file_name_for(file_1) %><br>
+            <strong>File name:</strong> <%= file_1_label %><br>
             <strong>File size:</strong> <%= file_size_for(file_1) %><br>
             <strong>Format:</strong>    <%= file_extension_for(file_1) %><br>
             <strong>Language:</strong>  <%= file_1_language %>
           </p>
-          <%= link_to file_1_label, file_1.url, title: file_1_label, target: '_blank', class: 'button' %>
+          <%= link_to "Download", file_1.url, title: "Download this file", target: '_blank', class: 'button' %>
```

Comment:
Do we want this to be the file label though? It could potentially be really long. Or do we want to throw something like `"Download: #{file_label}"` in there?